src/placement.ts

```diff
@@ -0,0 +1,5 @@
+export enum MraidPlacementType {
+  Unknown = "",
```

Comment:
According to the MRAID specs, return values can be either 'inline' (default value) or 'interstitial'.
I think we should not have an unknown type, and set the inline value as default.

Reply:
Placement type is reported later by SDK when it's loaded MRAID script. Placement is set in `mraid.notifyReady()`  just before triggering readiness of mraid object. Before that (by spec) it is okay to have not working methods, as mraid complaint ad should listen for readiness before doing anything.
I added Unknown state to explicitly say that placementType is not reported by SDK (name is self explanatory). We can return "inline" as default but returning empty string is more clearly indicates current state.
Technically we can even modify mraid.js when injecting it into container with proper placement type but it will greatly complicate the implementation.